README.md

```diff
@@ -40,11 +43,27 @@ This repo aims to provide a stripped down version of the default [WooCommerce E2
 "test:e2e-dev": "npx wc-e2e test:e2e-dev"
 ```
 
+* Create the file **`.nvmrc`** on your project root and add the content as **`v12`**.
```

Comment:
Just a thought - Would it be more convenient if we add `.nvmrc` to the package? The updated steps are fine, considering the possibility of an existing `.nvmrc` file.

Reply:
> Would it be more convenient if we add .nvmrc to the package? 

I think so.

Reply:
Added the `.nvmrc`.

Reply:
Since we added `.nvmrc`, this line can be removed.